content/kubernetes/active-active/preview/prepare-clusters.md

```diff
@@ -113,7 +113,7 @@ To communicate with other clusters, all participating clusters will need access
     type: Opaque
     ```
 
-1. Add the username and password new secret for that REC and namespace.
+1. Add the username and password to the new secret for that REC and namespace.
 
     This example shows the collected secrets file (`all-rec-secrets.yaml`) for `rec1` in namespace `ns1` and `rec2` in namespace `ns2`.
```

Comment:
it is no longer the convention of the rec name and ns.
suggestion:
This example shows the collected secrets file (all-rec-secrets.yaml) for rerc1 that represents the rec1 in namespace ns1 and rerc2 that represents the rec2 in namespace ns2.